src/models/Bookmark.ts

```diff
@@ -86,6 +86,11 @@ export const BookmarkModel = {
     { db }: Context
   ) {
     try {
+      // If there is a url to update and it does not have scheme, add http://
+      if (url && !url?.startsWith('http://') && !url?.startsWith('https://')) {
```

Comment:
thought: I wonder if we will need to make this something like `startsWith('.*://')` since a URL can start with other schemas. I don't think we need to since that seems like an unlikely use case. I'm sure we will find out if someone does something else.

Reply:
i had a similar thought, but wasn't sure if there was anything else we needed to support at the moment. agree that we can keep an eye on it and see if more use cases pop up.